squad/core/models.py

```diff
@@ -1104,7 +1104,7 @@ class TestSummary(TestSummaryBase):
 
     __test__ = False
 
-    def __init__(self, build, environment=None):
+    def __init__(self, build, environment=None, with_failures=True):
```

Comment:
not sure about this new conditional. maybe we should move collecting the failures to the same code that already calculates fixes and regressions (and with this scan every single in only one place)

The only place that actually seems to use this list of failures is the notification code, which will already also use fixes and regressions anyway.

Reply:
makes sense to me. I almost removed that a while ago, but @mwasilew pointed it out that notification emails might use that :)

Since the topic is hot, I'm trying to think a way of calculating regressions and fixes without having to rely on loading all tests into memory. If only there was a way to quickly get tests directly thru a hash or something...